src/backend/core/factories.py

```diff
@@ -148,6 +148,7 @@ class TeamFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = models.Team
         django_get_or_create = ("name",)
+        skip_postgeneration_save = True
```

Comment:
could you give more context on this one?

Reply:
I wanted to fix a warning occurring 124 times when running back end test suite : 
```
/usr/local/lib/python3.10/site-packages/factory/django.py:181: DeprecationWarning: TeamFactory._after_postgeneration will stop saving the instance after postgeneration hooks in the next major release.
  If the save call is extraneous, set skip_postgeneration_save=True in the TeamFactory.Meta.
  To keep saving the instance, move the save call to your postgeneration hooks or override _after_postgeneration.
    warnings.warn(
```